src/Microsoft.Diagnostics.Monitoring.StartupHook/Exceptions/CurrentAppDomainExceptionSource.cs

```diff
@@ -40,7 +41,10 @@ private void CurrentDomain_FirstChanceException(object? sender, FirstChanceExcep
             {
                 _handlingException.Value = true;
 
-                RaiseExceptionThrown(e.Exception, timestamp);
+                RaiseExceptionThrown(e.Exception,
+                    timestamp,
+                    Activity.Current?.Id ?? Guid.Empty.ToString(),
```

Comment:
These should probably be null if there is no activity information so that way we can differentiate between no activity vs custom activity identifiers.

Reply:
Do you want to preserve the `ActivityIdFormat` type as a nullable, or just convert it to its string value since that's what we ultimately present to the user?

Reply:
I think it would be best to propagate it as `ActivityIdFormat?` and let the UX decide how to treat that (e.g. `.ToString("G")` if not null).

Reply:
Discussed offline - can't use a nullable version of `ActivityIdFormat`, so keeping that as `Unknown` when not available.